frontends/ol-components/src/components/LearningResourceExpanded/LearningResourceExpanded.tsx

```diff
@@ -226,7 +236,14 @@ const CallToActionSection = ({
         size="large"
         href={getCallToActionUrl(resource) || ""}
       >
-        {cta}
+        {(platform?.code as PlatformEnum) === PlatformEnum.Ocw ? (
+          <>
+            Access Course Materials&nbsp;
+            <RiExternalLinkLine />
```

Comment:
StyledLink is a ButtonLink... It has a `endIcon` prop for putting icons after text. It spaces the icon (relative to the text) a bit better and also controls sizing. In this scenario, we should use `endIcon={ <RiExternalLinkLine />}`.

Reply:
forgot about the endIcon prop 👍